### PR TITLE
Various additions and changes to Swedish interface strings

### PR DESCRIPTION
**Pull request by phieri**

### CODE REVIEW

**Comment by petetnt** [comment]
Heja @Lominean! Could you sign the CLA? http://dev.brackets.io/brackets-contributor-license-agreement.html. Tack!

**Comment by Lominean** [comment]
CLA is signed.

**Comment by petetnt** [comment]
Tagging previous translators for a review: @robertkarlsson @tobimo @mikaeljorhult if you'd be interested in reviewing this, my swedish is kinda rusty 😸

File: src/nls/sv/strings.js [hunk 1]
```diff
@@ -370,11 +373,14 @@ define({
...
+    "CMD_TOGGLE_SIDEBAR"                  : "Växla på/av sidomeny",
+    "CMD_TOGGLE_PANELS"                   : "Växla på/av paneler",
+    "CMD_TOGGLE_PURE_CODE"                : "Inga distraktioner",
     "CMD_INCREASE_FONT_SIZE"              : "Öka textstorlek",
     "CMD_DECREASE_FONT_SIZE"              : "Minska teckenstorlek",
     "CMD_RESTORE_FONT_SIZE"               : "Återställ teckenstorlek",
-    "CMD_SCROLL_LINE_UP"                  : "Skrolla rad uppåt",
-    "CMD_SCROLL_LINE_DOWN"                : "Skrolla rad nedåt",
+    "CMD_SCROLL_LINE_UP"                  : "Rulla rad uppåt",
+    "CMD_SCROLL_LINE_DOWN"                : "Rulla rad nedåt",
```
Comment (robertkarlsson) [inline src/nls/sv/strings.js]: Rulla is not really a word used by Swedish programmers in this context. It's correct translated, but I've never heard anyone use it. Skrolla/scrolla would be easier to understand, though it's "Swenglish"

File: src/nls/sv/strings.js [hunk 2]
```diff
@@ -550,6 +571,8 @@ define({
     "EXTENSIONS_AVAILABLE_TITLE"           : "Tillgängliga",
     "EXTENSIONS_THEMES_TITLE"              : "Teman",
     "EXTENSIONS_UPDATES_TITLE"             : "Uppdateringar",
+    "EXTENSIONS_LAST_UPDATED"              : "Senast uppdaterad",
+    "EXTENSIONS_DOWNLOADS"                 : "Nerladdningar",
```
Comment (robertkarlsson) [inline src/nls/sv/strings.js]: The correct Swedish here is "Nerladdningar" -> "Nedladdningar"

**Review by robertkarlsson — COMMENTED** [review]
I think it looks good overall. Not sure about "skrolla" -> "rulla". Anybody else have an opinion about this?

File: src/nls/sv/strings.js (hunk 1 above)
Comment (Lominean) [inline src/nls/sv/strings.js]: Based on [recommendation by Datatermgruppen](http://www.datatermgruppen.se/ordlista.html#b95) and my own preference. But I agree that "skrolla" is perfectly understandable.

**Review by petetnt — APPROVED** [review]
LGTM

**Pull request merged**